### PR TITLE
Site Editor: Unregister site editor feedback plugin

### PR DESCRIPTION
**Pull request by vykes-mac**
### Changes proposed in this Pull Request

* remove site editor feedback form 

#### Before 
image
#### After 
image

### Testing instructions

- Checkout branch
- Sandbox widgets.wp.com
- Sync wpcom-block-editor changes to sandbox PCYsg-l4k-p2
- Navigate to `calypso/apps/wpcom-block-editor` in terminal
- Execute `yarn dev --sync`
- Verify that the feedback plugin is no longer present in the site editor

Related to #61774

### CODE REVIEW

File: apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js [hunk 1]
```diff
@@ -980,25 +979,6 @@ async function preselectParentPage() {
...
-		registerPlugin( 'a8c-fse-beta-feedback-plugin', {
-			render: () => (
-				<PluginSidebar
-					name="a8c-fse-beta-feedback-plugin"
-					title={ __( 'Site Editor Beta Feedback', 'full-site-editing' ) }
-					// eslint-disable-next-line wpcalypso/jsx-classname-namespace
-					className="a8c-site-editor-feedback-plugin"
-					icon={ comment }
-				>
-					<FeedbackForm calypsoPort={ calypsoPort } />
```
Comment (zaguiini) [inline apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js:995]: This component is not used anywhere else, so we might as well remove its file.

**Review by zaguiini — COMMENTED** [review]
Great stuff 👍

File: apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js (hunk 1 above)
Comment (p-jackson) [reply apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js:995]: Nice catch, removing that will allow us to tidy up a few other things, like the handling for the `'sendSiteEditorBetaFeedback'` iframe message too 👍

**Comment by Addison-Stavlo** [comment]
Looking good so far!

I think keeping this PR separate from the other side of the Iframe will be a good deployment strategy.  As we will want to remove the registered plugin (as done here) and deploy that wpcom-block-editor change for a day or so before we rip out the calypso side (atomic sites can take up to a day to see wpcom-block-editor changes unless we go about some special systems cache busting iirc).

File: apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js (hunk 1 above)
Comment (vykes-mac) [reply apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js:995]: > I think keeping this PR separate from the other side of the Iframe will be a good deployment strategy. As we will want to remove the registered plugin (as done here) and deploy that wpcom-block-editor change for a day or so before we rip out the calypso side (atomic sites can take up to a day to see wpcom-block-editor changes unless we go about some special systems cache busting iirc).

Can you please clarify?

Comment (vykes-mac) [reply apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js:995]: @p-jackson are you referring to those references in `client/gutenberg/editor/calypsoify-iframe.tsx`?

Comment (p-jackson) [reply apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js:995]: Yes, that's right. We won't need the code in `calypsoify-iframe.tsx` that handles this message once there's no code that sends the message. But @Addison-Stavlo makes a good point that we need to be thoughtful about our deployment strategy. We'll want to remove the code for _sending_ the message in one deployment (including removing `feedback-form.js` which you've done), and then remove the code for _handling_ the message in a follow up deploy.

So a separate PR for the changes to `calypsoify-iframe.tsx` changes.

**Review by p-jackson — APPROVED** [review]
Works just as expected, no more feedback UI.

**Review by zaguiini — APPROVED** [review]

**Pull request merged**